GliaWidgets/Localization2/Localization2+Live.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+extension Localization2 {
+    static func live(stringProviding: @escaping () -> StringProvidingPhase?) -> Self {
+        .init(callVisualizer: .init(
+            screenSharing: .init(
+                message: Localization2.tr(
```

Comment:
So on the final version, this will look more like `message: Localization.ScreenSharing.message` rather than this? Because if the `tr` itself is here, we lose the autogenerated code that gets generated from the strings file, which would mean duplication and we would be fixing a problem by creating another.

Reply:
Autogeneration in the old fashion wasn't the envisioned outcome.

Reply:
No final version looks exactly like it is implemented here

Reply:
Okay, if everyone else is fine with this approach, then I will not complain :) I find it a huge detriment though to manage this manually.